.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,47 @@
+name: Feature request
+description:  |
```

Comment:
Add title and labels tag as mentioned above
`title: "[Feature Request] <Title>"`
`labels: [feature request]`

![image](https://user-images.githubusercontent.com/47073516/156115519-fd2c72d2-7153-4d74-adb7-be30140b0a8b.png)